Automattic/woocommerce-payments
# Add dismissable Notice to message the rename of Deposits to Payout

Fixes #9673

#### PENDING
- ~get final illustration and copy  - [Ref](https://github.com/Automattic/woocommerce-payments/issues/9673#issuecomment-2458287089) [[final copy provided](https://github.com/Automattic/woocommerce-payments/issues/9673#issuecomment-2465232200)]~
- ~tests~

#### Changes proposed in this Pull Request
Adds a one-time dismissible spotlight notice to the Payouts page when user opens either the [payments overview page, deposits list page, or deposit details page](https://github.com/Automattic/woocommerce-payments/issues/9673#issuecomment-2462835511).


Testing instructions

- Make sure option `wcpay_payouts_rename_notice_dismissed` doesn't exist either by running `delete_option( 'wcpay_payouts_rename_notice_dismissed' )` on [WP Console](https://wordpress.org/plugins/wp-console/) or delete it from your `wp_options` table.
- Open either payments overview page, deposits list page, or deposit details page.
- Confirm that you will see the spotlight notice and make sure copy is [as provided](https://github.com/Automattic/woocommerce-payments/issues/9673#issuecomment-2465232200).

![Screenshot 2024-11-09 at 04 10 20](https://github.com/user-attachments/assets/9b4aab7e-8a4d-4205-80ce-a7732a11d93e)

- Make sure when you dismiss the notice, it doesn't appear again on any of the pages reload.

-------------------


**Post merge**
